pyciemss/interfaces.py

```diff
@@ -771,8 +771,12 @@ def optimize(
     logging_step_size: float,
     qoi: Callable,
     risk_bound: float,
-    static_parameter_interventions: Callable[
-        [torch.Tensor], Dict[float, Dict[str, Intervention]]
+    static_parameter_interventions: Union[
```

Comment:
This is fine. If you want you can make this a bit more concise with a type alias.

E.g. https://github.com/BasisResearch/chirho/blob/master/chirho/dynamical/ops.py#L14

Here, you could add a line with the following:

```
InterventionFunc = Callable[[torch.Tensor], Dict[float, Dict[str, Intervention]]]
```

Then this type could be:

```
Union[InterventionFunc, Callable[[torch.Tensor], List[InterventionFunc]]
```

Reply:
Looking again, is this type signature correct? Should it be:

```
Callable[[torch.Tensor], Dict[float, Dict[str, Intervention]]],
        Callable[
            [torch.Tensor],
            List[Dict[float, Dict[str, Intervention]]],
        ]
```

Reply:
I am not sure. It might be. I was thinking once I create the lambda function it becomes a Callable of Callable functions.. Let's discuss when we meet.